public/javascript/global.js

```diff
@@ -41,12 +41,12 @@ function populateTable() {
     });
 };
 
-$("form")
-    .submit(function(e) {
-        var path = document.forms['uploadFile'];
-        console.log(JSON.stringify(path));
-        if (path === undefined) {
-            displayError("Please select a file.");
+$("form").submit(function(e) {
+        e.preventDefault();
+        var path = $(".fileUpload")[0].files;
+        console.log(path);
+        if (path.length < 1) {
```

Comment:
path will have an object containing the files, if it's empty, the object has zero length, not undefined.